guide/latest/feature-sosreport.html

```diff
@@ -25,7 +25,7 @@
 <div class="titlepage"><div><div><h2 class="title">
 <a name="feature-sosreport"></a>SOS Report</h2></div></div></div>
 <p>If present on the system Cockpit can use <code class="code">sosreport</code> to collect system configuration and diagnostic information.</p>
-<p>If <code class="code">sosreport</code> is not found on an Atomic system, cockpit will attempt to run <code class="code">sosreport</code> in the <code class="code">rhel7/support-tools</code> docker container.</p>
+<p>If <code class="code">sosreport</code> is not found on a system, cockpit will attempt to run <code class="code">sosreport</code> in the <code class="code">rhel7/support-tools</code> docker container.</p>
```

Comment:
This bit needs to be dropped from this PR, as it gets auto-synced from cockpit: https://github.com/cockpit-project/cockpit/blob/master/doc/guide/feature-sosreport.xml I'm happy to do that myself tomorrow.